index.js

```diff
@@ -31,17 +31,18 @@ module.exports = (fn, options) => {
 		cachePromiseRejection: false
 	}, options);
 
-	if (typeof options.maxAge === 'number') {
+	if (typeof options.maxAge === 'number' && options.maxAge > 0) {
```

Comment:
If the user specifies `{maxAge: 0}` we have a memory leak as it will never clean up those entries.

Reply:
I assumed the intended behavior of `{maxAge: 0}` would be to disable age-based eviction. Why would someone memoize with `{maxAge: 0}`? I'm happy to change the condition to `options.maxAge >= 0`

Reply:
I assumed the intended behavior of `{maxAge: 0}` would be to disable age-based eviction. Why would someone memoize with `{maxAge: 0}`? I'm happy to change the condition to `options.maxAge >= 0`

Reply:
> I assumed the intended behavior of {maxAge: 0} would be to disable age-based eviction.

Assumptions are dangerous. Someone could also assume it causes it to be evicted right away.

> Why would someone memoize with {maxAge: 0}?

I dunno, but from experience, users do a lot of things you don't assume they would do.

>  I'm happy to change the condition to options.maxAge >= 0

👍 Also document what setting `0` does and I think we should also document and validate that the number is non-negative, to prevent mistakes.

Reply:
Done